assets/components/headers/countrySwitcherHeader/countrySwitcherHeader.jsx

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import { SvgGuardianLogo } from 'components/svg/svg';
+import CountryGroupSwitcher from 'components/countryGroupSwitcher/countryGroupSwitcher';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+
+type PropTypes = {
+  countryGroupIds: CountryGroupId[],
+  selectedCountryGroup: CountryGroupId,
+  onCountryGroupSelect: (string) => void,
+};
```

Comment:
If these are identical to the `CountryGroupSwitcher`, and are just being passed through to it, you could wrap that component in a container, and have this one take it as a `prop` or as `children`. That would reduce the number of props being passed through components that don't care about them, and possibly reduce the amount of computation on each prop change.

Reply:
Discussed offline, we decided it wasn't worth it for this component, it's quite simple and doing this may increase the complexity.